### PR TITLE
Ros2 fix topic name part1

### PR DESCRIPTION
**Pull request by isamu-takagi**
Fixed some topic names to match ros1.
Related PR: https://github.com/tier4/autoware_launcher.iv.universe/pull/83

### CODE REVIEW

File: planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp [hunk 1]
```diff
@@ -52,31 +52,31 @@ ObstacleAvoidancePlanner::ObstacleAvoidancePlanner()
...
 
-  trajectory_pub_ = create_publisher<autoware_planning_msgs::msg::Trajectory>("output/path", 1);
+  trajectory_pub_ = create_publisher<autoware_planning_msgs::msg::Trajectory>("~/output/path", 1);
   avoiding_traj_pub_ = create_publisher<autoware_planning_msgs::msg::Trajectory>(
     "/planning/scenario_planning/lane_driving/obstacle_avoidance_candidate_trajectory",
     durable_qos);
-  debug_smoothed_points_pub_ =
-    create_publisher<autoware_planning_msgs::msg::Trajectory>("debug/smoothed_poins", durable_qos);
+  debug_smoothed_points_pub_ = create_publisher<autoware_planning_msgs::msg::Trajectory>(
+    "~/debug/smoothed_poins", durable_qos);
```
Comment (wep21) [inline planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp:60]: typo?"poins"->"points" cc @k0suke-murakami

**Comment by wep21** [comment]
```
/planning/mission_planning/checkpoint
/planning/mission_planning/echo_back_goal_pose
/planning/mission_planning/goal
/planning/mission_planning/route
/planning/mission_planning/route_marker
/planning/scenario_planning/current_max_velocity
/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/blind_spot
/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/crosswalk
/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/detection_area
/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/intersection
/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/path
/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/stop_line
/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/traffic_light
/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/input/external_crosswalk_states
/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/input/external_intersection_states
/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/output/stop_reason
/planning/scenario_planning/lane_driving/behavior_planning/debug/traffic_light_state
/planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/debug/drivable_area
/planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/debug/predicted_path_markers
/planning/scenario_planning/lane_driving/behavior_planning/path
/planning/scenario_planning/lane_driving/behavior_planning/path_with_lane_id
/planning/scenario_planning/lane_driving/force_lane_change
/planning/scenario_planning/lane_driving/lane_change_approval
/planning/scenario_planning/lane_driving/lane_change_available
/planning/scenario_planning/lane_driving/lane_change_candidate_path
/planning/scenario_planning/lane_driving/lane_change_ready
/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/area_with_objects
/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/clearance_map
/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/marker
/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/object_clearance_map
/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/smoothed_poins
/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/trajectory
/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/debug/marker
/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/debug_values
/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/input/expand_stop_range
/planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/debug/marker
/planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/trajectory
/planning/scenario_planning/lane_driving/obstacle_avoidance_approval
/planning/scenario_planning/lane_driving/obstacle_avoidance_candidate_trajectory
/planning/scenario_planning/lane_driving/obstacle_avoidance_ready
/planning/scenario_planning/lane_driving/trajectory
/planning/scenario_planning/max_velocity
/planning/scenario_planning/motion_velocity_optimizer/closest_acceleration
/planning/scenario_planning/motion_velocity_optimizer/closest_jerk
/planning/scenario_planning/motion_velocity_optimizer/closest_velocity
/planning/scenario_planning/motion_velocity_optimizer/debug/trajectory_external_velocity_limited
/planning/scenario_planning/motion_velocity_optimizer/debug/trajectory_lateral_acc_filtered
/planning/scenario_planning/motion_velocity_optimizer/debug/trajectory_raw
/planning/scenario_planning/motion_velocity_optimizer/debug/trajectory_time_resampled
/planning/scenario_planning/motion_velocity_optimizer/distance_to_stopline
/planning/scenario_planning/motion_velocity_optimizer/stop_speed_exceeded
/planning/scenario_planning/parking/costmap_generator/grid_map
/planning/scenario_planning/parking/costmap_generator/occupancy_grid
/planning/scenario_planning/parking/freespace_planner/debug/partial_pose_array
/planning/scenario_planning/parking/freespace_planner/debug/pose_array
/planning/scenario_planning/parking/trajectory
/planning/scenario_planning/scenario
/planning/scenario_planning/scenario_selector/trajectory
/planning/scenario_planning/status/no_start_reason
/planning/scenario_planning/status/stop_reason
/planning/scenario_planning/status/stop_reasons
/planning/scenario_planning/trajectory
/system/emergency/control_cmd
/system/emergency/hazard_status
/system/emergency/is_emergency
/system/emergency/shift_cmd
/system/emergency/turn_signal_cmd
/system/state_timeout_checker/is_state_timeout
```

**Comment by wep21** [comment]
```
Node [/control/trajectory_follower/mpc_follower]
Publications: 
 * /control/trajectory_follower/lateral/control_cmd [autoware_control_msgs/ControlCommandStamped]
 * /control/trajectory_follower/mpc_follower/debug/debug_values [std_msgs/Float32MultiArray]
 * /control/trajectory_follower/mpc_follower/debug/markers [visualization_msgs/MarkerArray]
 * /control/trajectory_follower/mpc_follower/debug/mpc_calc_time [std_msgs/Float32]
 * /control/trajectory_follower/mpc_follower/parameter_descriptions [dynamic_reconfigure/ConfigDescription]
 * /control/trajectory_follower/mpc_follower/parameter_updates [dynamic_reconfigure/Config]
 * /control/trajectory_follower/predicted_trajectory [autoware_planning_msgs/Trajectory]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /localization/twist [geometry_msgs/TwistStamped]
 * /planning/scenario_planning/trajectory [autoware_planning_msgs/Trajectory]
 * /tf [tf2_msgs/TFMessage]
 * /tf_static [tf2_msgs/TFMessage]
 * /vehicle/status/steering [autoware_vehicle_msgs/Steering]

Node [/control/trajectory_follower/lane_departure_checker_node]
Publications: 
 * /control/trajectory_follower/lane_departure_checker_node/debug/deviation/lateral [std_msgs/Float64]
 * /control/trajectory_follower/lane_departure_checker_node/debug/deviation/yaw [std_msgs/Float64]
 * /control/trajectory_follower/lane_departure_checker_node/debug/deviation/yaw_deg [std_msgs/Float64]
 * /control/trajectory_follower/lane_departure_checker_node/debug/marker_array [visualization_msgs/MarkerArray]
 * /control/trajectory_follower/lane_departure_checker_node/debug/processing_time_ms [diagnostic_msgs/DiagnosticStatus]
 * /control/trajectory_follower/lane_departure_checker_node/parameter_descriptions [dynamic_reconfigure/ConfigDescription]
 * /control/trajectory_follower/lane_departure_checker_node/parameter_updates [dynamic_reconfigure/Config]
 * /diagnostics [diagnostic_msgs/DiagnosticArray]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /control/trajectory_follower/predicted_trajectory [autoware_planning_msgs/Trajectory]
 * /localization/twist [geometry_msgs/TwistStamped]
 * /map/vector_map [autoware_lanelet2_msgs/MapBin]
 * /planning/mission_planning/route [autoware_planning_msgs/Route]
 * /planning/scenario_planning/trajectory [autoware_planning_msgs/Trajectory]
 * /tf [tf2_msgs/TFMessage]
 * /tf_static [tf2_msgs/TFMessage]

Node [/control/trajectory_follower/velocity_controller]
Publications: 
 * /control/trajectory_follower/longitudinal/control_cmd [autoware_control_msgs/ControlCommandStamped]
 * /control/trajectory_follower/velocity_controller/debug_values [std_msgs/Float32MultiArray]
 * /control/trajectory_follower/velocity_controller/parameter_descriptions [dynamic_reconfigure/ConfigDescription]
 * /control/trajectory_follower/velocity_controller/parameter_updates [dynamic_reconfigure/Config]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /localization/twist [geometry_msgs/TwistStamped]
 * /planning/scenario_planning/trajectory [autoware_planning_msgs/Trajectory]
 * /tf [tf2_msgs/TFMessage]
 * /tf_static [tf2_msgs/TFMessage]

Node [/planning/scenario_planning/motion_velocity_optimizer]
Publications: 
 * /planning/scenario_planning/current_max_velocity [std_msgs/Float32]
 * /planning/scenario_planning/motion_velocity_optimizer/closest_acceleration [std_msgs/Float32]
 * /planning/scenario_planning/motion_velocity_optimizer/closest_jerk [std_msgs/Float32]
 * /planning/scenario_planning/motion_velocity_optimizer/closest_velocity [std_msgs/Float32]
 * /planning/scenario_planning/motion_velocity_optimizer/debug/trajectory_external_velocity_limited [autoware_planning_msgs/Trajectory]
 * /planning/scenario_planning/motion_velocity_optimizer/debug/trajectory_lateral_acc_filtered [autoware_planning_msgs/Trajectory]
 * /planning/scenario_planning/motion_velocity_optimizer/debug/trajectory_raw [autoware_planning_msgs/Trajectory]
 * /planning/scenario_planning/motion_velocity_optimizer/debug/trajectory_time_resampled [autoware_planning_msgs/Trajectory]
 * /planning/scenario_planning/motion_velocity_optimizer/distance_to_stopline [std_msgs/Float32]
 * /planning/scenario_planning/motion_velocity_optimizer/parameter_descriptions [dynamic_reconfigure/ConfigDescription]
 * /planning/scenario_planning/motion_velocity_optimizer/parameter_updates [dynamic_reconfigure/Config]
 * /planning/scenario_planning/motion_velocity_optimizer/stop_speed_exceeded [std_msgs/Bool]
 * /planning/scenario_planning/trajectory [autoware_planning_msgs/Trajectory]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /localization/twist [geometry_msgs/TwistStamped]
 * /planning/scenario_planning/max_velocity [std_msgs/Float32]
 * /planning/scenario_planning/scenario_selector/trajectory [autoware_planning_msgs/Trajectory]
 * /tf [tf2_msgs/TFMessage]
 * /tf_static [tf2_msgs/TFMessage]

Node [/planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner]
Publications: 
 * /planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/debug/drivable_area [nav_msgs/OccupancyGrid]
 * /planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/debug/predicted_path_markers [visualization_msgs/MarkerArray]
 * /planning/scenario_planning/lane_driving/behavior_planning/path_with_lane_id [autoware_planning_msgs/PathWithLaneId]
 * /planning/scenario_planning/lane_driving/lane_change_available [std_msgs/Bool]
 * /planning/scenario_planning/lane_driving/lane_change_candidate_path [autoware_planning_msgs/Path]
 * /planning/scenario_planning/lane_driving/lane_change_ready [std_msgs/Bool]
 * /planning/scenario_planning/status/stop_reasons [autoware_planning_msgs/StopReasonArray]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /localization/twist [geometry_msgs/TwistStamped]
 * /map/vector_map [autoware_lanelet2_msgs/MapBin]
 * /perception/object_recognition/objects [autoware_perception_msgs/DynamicObjectArray]
 * /planning/mission_planning/route [autoware_planning_msgs/Route]
 * /planning/scenario_planning/lane_driving/force_lane_change [unknown type]
 * /planning/scenario_planning/lane_driving/lane_change_approval [std_msgs/Bool]
 * /tf [tf2_msgs/TFMessage]
 * /tf_static [tf2_msgs/TFMessage]
 
Node [/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner]
Publications: 
 * /planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/blind_spot [visualization_msgs/MarkerArray]
 * /planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/crosswalk [visualization_msgs/MarkerArray]
 * /planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/detection_area [visualization_msgs/MarkerArray]
 * /planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/intersection [visualization_msgs/MarkerArray]
 * /planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/path [visualization_msgs/MarkerArray]
 * /planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/stop_line [visualization_msgs/MarkerArray]
 * /planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/debug/traffic_light [visualization_msgs/MarkerArray]
 * /planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/output/stop_reason [diagnostic_msgs/DiagnosticStatus]
 * /planning/scenario_planning/lane_driving/behavior_planning/debug/traffic_light_state [autoware_perception_msgs/TrafficLightStateStamped]
 * /planning/scenario_planning/lane_driving/behavior_planning/path [autoware_planning_msgs/Path]
 * /planning/scenario_planning/status/stop_reasons [autoware_planning_msgs/StopReasonArray]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /foa/traffic_light_recognition/traffic_light_states [unknown type]
 * /localization/twist [geometry_msgs/TwistStamped]
 * /map/vector_map [autoware_lanelet2_msgs/MapBin]
 * /perception/object_recognition/objects [autoware_perception_msgs/DynamicObjectArray]
 * /perception/traffic_light_recognition/traffic_light_states [autoware_perception_msgs/TrafficLightStateArray]
 * /planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/input/external_crosswalk_states [unknown type]
 * /planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/input/external_intersection_states [unknown type]
 * /planning/scenario_planning/lane_driving/behavior_planning/path_with_lane_id [autoware_planning_msgs/PathWithLaneId]
 * /sensing/lidar/no_ground/pointcloud [sensor_msgs/PointCloud2]
 * /tf [tf2_msgs/TFMessage]
 * /tf_static [tf2_msgs/TFMessage]

Node [/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner]
Publications: 
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/area_with_objects [nav_msgs/OccupancyGrid]
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/clearance_map [nav_msgs/OccupancyGrid]
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/marker [visualization_msgs/MarkerArray]
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/object_clearance_map [nav_msgs/OccupancyGrid]
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/smoothed_poins [autoware_planning_msgs/Trajectory]
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/parameter_descriptions [dynamic_reconfigure/ConfigDescription]
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/parameter_updates [dynamic_reconfigure/Config]
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/trajectory [autoware_planning_msgs/Trajectory]
 * /planning/scenario_planning/lane_driving/obstacle_avoidance_candidate_trajectory [autoware_planning_msgs/Trajectory]
 * /planning/scenario_planning/lane_driving/obstacle_avoidance_ready [std_msgs/Bool]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /localization/twist [geometry_msgs/TwistStamped]
 * /perception/object_recognition/objects [autoware_perception_msgs/DynamicObjectArray]
 * /planning/scenario_planning/lane_driving/behavior_planning/path [autoware_planning_msgs/Path]
 * /planning/scenario_planning/lane_driving/obstacle_avoidance_approval [unknown type]
 * /tf [tf2_msgs/TFMessage]
 * /tf_static [tf2_msgs/TFMessage]

Node [/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner]
Publications: 
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/debug/marker [visualization_msgs/MarkerArray]
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/debug_values [std_msgs/Float32MultiArray]
 * /planning/scenario_planning/lane_driving/trajectory [autoware_planning_msgs/Trajectory]
 * /planning/scenario_planning/status/stop_reason [diagnostic_msgs/DiagnosticStatus]
 * /planning/scenario_planning/status/stop_reasons [autoware_planning_msgs/StopReasonArray]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /localization/twist [geometry_msgs/TwistStamped]
 * /perception/object_recognition/objects [autoware_perception_msgs/DynamicObjectArray]
 * /planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/input/expand_stop_range [unknown type]
 * /planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/trajectory [autoware_planning_msgs/Trajectory]
 * /sensing/lidar/no_ground/pointcloud [sensor_msgs/PointCloud2]
 * /tf [tf2_msgs/TFMessage]
 * /tf_static [tf2_msgs/TFMessage]
 
Node [/planning/scenario_planning/parking/costmap_generator]
Publications: 
 * /planning/scenario_planning/parking/costmap_generator/grid_map [grid_map_msgs/GridMap]
 * /planning/scenario_planning/parking/costmap_generator/occupancy_grid [nav_msgs/OccupancyGrid]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /map/vector_map [autoware_lanelet2_msgs/MapBin]
 * /perception/object_recognition/objects [autoware_perception_msgs/DynamicObjectArray]
 * /planning/scenario_planning/scenario [autoware_planning_msgs/Scenario]
 * /sensing/lidar/no_ground/pointcloud [sensor_msgs/PointCloud2]
 * /tf [tf2_msgs/TFMessage]
 * /tf_static [tf2_msgs/TFMessage]

Node [/planning/scenario_planning/parking/freespace_planner]
Publications: 
 * /planning/scenario_planning/parking/freespace_planner/debug/partial_pose_array [geometry_msgs/PoseArray]
 * /planning/scenario_planning/parking/freespace_planner/debug/pose_array [geometry_msgs/PoseArray]
 * /planning/scenario_planning/parking/trajectory [autoware_planning_msgs/Trajectory]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /localization/twist [geometry_msgs/TwistStamped]
 * /planning/mission_planning/route [autoware_planning_msgs/Route]
 * /planning/scenario_planning/parking/costmap_generator/occupancy_grid [nav_msgs/OccupancyGrid]
 * /planning/scenario_planning/scenario [autoware_planning_msgs/Scenario]
 * /tf [tf2_msgs/TFMessage]
 * /tf_static [tf2_msgs/TFMessage]

Node [/system/emergency_handler]
Publications: 
 * /diagnostics_err [diagnostic_msgs/DiagnosticArray]
 * /rosout [rosgraph_msgs/Log]
 * /system/emergency/control_cmd [autoware_control_msgs/ControlCommandStamped]
 * /system/emergency/hazard_status [autoware_system_msgs/HazardStatusStamped]
 * /system/emergency/is_emergency [std_msgs/Bool]
 * /system/emergency/shift_cmd [autoware_vehicle_msgs/ShiftStamped]
 * /system/emergency/turn_signal_cmd [autoware_vehicle_msgs/TurnSignal]

Subscriptions: 
 * /autoware/driving_capability [autoware_system_msgs/DrivingCapability]
 * /autoware/state [autoware_system_msgs/AutowareState]
 * /control/current_gate_mode [autoware_control_msgs/GateMode]
 * /control/vehicle_cmd [autoware_vehicle_msgs/VehicleCommand]
 * /localization/twist [geometry_msgs/TwistStamped]
 * /system/state_timeout_checker/is_state_timeout [std_msgs/Bool]

**Comment by wep21** [comment]
@isamu-takagi Could you fix typo and resolve a conflict?

File: planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp (hunk 1 above)
Comment (isamu-takagi) [reply planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp]: Fixed.

**Review by wep21 — APPROVED** [review]
LGTM

**Pull request merged**